mmv1/templates/terraform/resource_definition/firebase_project_location_deprecation.go.erb

```diff
@@ -0,0 +1,17 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2023 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+DeprecationMessage: "Deprecated. Specify a location directly in google_app_engine_application " +
```

Comment:
I think leaving the deprecation message similar to this is probably fine since it would reach folks setting up the resource for the first time. Possibly something like:

> Instead of using google_firebase_project_location, explicitly configure `google_app_engine_application.location_id` and `google_firestore_database.location_id` instead

Reply:
Done.